build.psm1

```diff
@@ -907,7 +907,8 @@ function Start-PSPester {
         [switch]$IncludeCommonTests,
         [string]$ExperimentalFeatureName,
         [Parameter(HelpMessage='Title to publish the results as.')]
-        [string]$Title = 'PowerShell Core Tests'
+        [string]$Title = 'PowerShell Core Tests',
+        [switch]$Wait
```

Comment:
Why we add this in the PR?

Reply:
I needed this change to debug this issue